src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
+import Calculator from './components/Calculator';
+import './styles/App.css';
+import Header from './components/Header';
+
+/* import React, { useState } from 'react'; */
+/* <nav className={styles.nav}></nav> */
+const App = () => (
```

Comment:
- [ ] Let's try not to have commented unused codes from our codebase as they make our codebase look dirty and unprofessional. Kindly remove all unused commented codes from your codebase so your code looks clean and professional 👍🏽

Reply:
This change has been done well